### PR TITLE
Traidor mission in new category and 8h timeout

### PR DESCRIPTION
**Pull request by aefreedman**
moved traidor mission into new mission category "AS_T" and added a 28800 tick (8H) timeout

### CODE REVIEW

**Comment by aefreedman** [comment]
oh wait there's weighting in the mission chances now hold up

File: missions/as_traidor.sqf [hunk 1]
```diff
@@ -166,7 +166,7 @@ else
 		};
 	};
 
-_nul = [1200,_tsk] spawn borrarTask;
+_nul = [28800,_tsk] spawn borrarTask;
```
Comment (nateberkopec) [inline missions/as_traidor.sqf:169]: I don't like this change. I think the mission timeout is good as-is.

Comment (nateberkopec) [reply missions/as_traidor.sqf:169]: Er, actually, I read what the forum post said, but Borrartask seems to be a timeout for how long the mission stays active, not the timeout *between* missions.

**Comment by nateberkopec** [comment]
Closing in favor of #3, we can fix the timeout separately.

**Pull request closed without merging**